lib/terraorg/model/org.rb

```diff
@@ -264,7 +269,17 @@ def generate_tf
     all_managers = Set.new([@manager] + @platoons.all.map(&:manager) + @squads.all.map(&:manager).select { |m| m })
     manager_dl = "#{@id}-managers"
     tf += Util.gsuite_group_tf(manager_dl, @gsuite_domain, all_managers, "All managers of the #{@name} organization (terraorg)")
+    tf
+  end
+
+  def generate_tf
+    tf = generate_tf_platoons
+    File.write('auto.platoons.tf', tf)
+
+    tf = generate_tf_sqauds
```

Comment:
* The methods seem commented out and this doesn't look like it works?
* `sqauds` should be `squads` in any case

Reply:
fixed